.github/workflows/ci.yml

```diff
@@ -78,7 +78,7 @@ jobs:
       - name: Build and run stack
         run: |
           printenv
-          docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build -d worker
+          docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build --build-arg GIT_COMMIT=$(git rev-parse --short=10 HEAD) -d worker
```

Comment:
Instead of using a bash command to get the commit hash you can use the `github.sha` from the github context https://docs.github.com/en/actions/learn-github-actions/contexts#github-context if that works for you

Reply:
Perfect, that's what I was looking for.  Webpack worked fine locally with the `.git` directory, but then of course docker build doesn't have access to that..